docs/src/api.md

```diff
@@ -40,6 +40,17 @@ boot = parametricbootstrap(MersenneTwister(42), 1000, fm1)
 coefplot(boot; conf_level=0.999, title="Custom Title")
 ```
 
+## Ridge Plots
+
+```@docs
+ridgeplot
+```
+
+```@example Coefplot
```

Comment:
```suggestion
```@example Ridgeplot
```

Reply:
That's intentional: reusing the name there means that I inherit the objects from the previous `Coefplot` blocks (and in particular: the computed bootstrap `boot`).